dask_sql/context.py

```diff
@@ -690,14 +689,7 @@ def ipython_magic(
             self, auto_include=auto_include, disable_highlighting=disable_highlighting
         )
 
-    def run_server(
-        self,
-        client: Client = None,
-        host: str = "0.0.0.0",
-        port: int = 8080,
-        log_level=None,
-        blocking: bool = True,
-    ):  # pragma: no cover
+    def run_server(self, **kwargs):  # pragma: no cover
```

Comment:
Made this change because currently it isn't possible to do something like

```python
c = Context()
c.create_table("df", timeseries())
c.run_server(jdbc_metadata=True)
```

Think it might also make sense to use the same docstring for `Context.run_server` and `app.run_server` so that they're kept in sync (currently `Context.run_server` is outdated)